.github/workflows/owasp.yaml

```
@@ -41,6 +41,7 @@ jobs:
         env:
           JAVA_HOME: /opt/jdk
       - name: Upload Test results
+        if: ${{ always() }}
```

```suggestion
        if: always()
```

This can be simplified to `if: always()`